src/main/java/ua/com/juja/microservices/teams/slackbot/service/impl/TeamServiceImpl.java

```diff
@@ -41,27 +43,45 @@ public TeamServiceImpl(TeamRepository teamRepository, UserService userService) {
     }
 
     @Override
-    public Team activateTeam(String text) {
+    public Team activateTeam(String fromUser, String text) {
         Utils.checkNull(text, "Text must not be null!");
-        List<String> slackNames = SlackNameHandler.getSlackNamesFromText(text);
+        Utils.checkNull(fromUser, "FromUser must not be null!");
+        Set<String> slackNames = SlackNameHandler.getSlackNamesFromText(text);
         if (slackNames.size() != TEAM_SIZE) {
             throw new WrongCommandFormatException(String.format("We found %d slack names in your command." +
                     " But size of the team must be %s.", slackNames.size(), TEAM_SIZE));
         }
-        Set<User> users = new HashSet<>(userService.findUsersBySlackNames(slackNames));
-        if (users.size() != slackNames.size()) {
-            throwUserExchangeException();
-        }
-        Set<String> uuids = users.stream()
-                .map(User::getUuid)
-                .collect(Collectors.toSet());
-        TeamRequest teamRequest = new TeamRequest(uuids);
-        Team activatedTeam = teamRepository.activateTeam(teamRequest);
-        checkTeamMembersEquality(teamRequest.getMembers(), activatedTeam.getMembers());
+        fromUser = SlackNameHandler.addAtToSlackName(fromUser);
```

Comment:
Could you think about this code?
Is this code simple for understanding and reading?
Could you change it for simple reading and understanding?
Maybe refactoring and some method extraction help you.
Could you try to do it?